test/fixtures/cloudbuild_enabled/main.tf

```diff
@@ -14,13 +14,22 @@
  * limitations under the License.
  */
 
+resource "random_string" "suffix" {
```

Comment:
I think this should be a toggleable variable for the module, so we can keep the the module interface minimal and people can accept the default names / randomization if they wish or override the behaviour.